### PR TITLE
Add Rollbar to s3 zipper, and make it panic if there is a db initialization failure

### PR DESCRIPTION
**Pull request by xxx**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,12 +6,12 @@ services:
       context: .
     network_mode: host
     ports:
-      - '8000:8000'
+      - '8000:7766'
```
Comment (xxx) [inline docker-compose.yml:9]: there's a conflict if it's run on port 8000 in the container, and it's not clear why that happens. Normally this isn't an issue.

File: main.go [hunk 2]
```diff
@@ -2,39 +2,44 @@ package main
 
 import (
```
Comment (xxx) [inline main.go:3]: a lot of the changes in this file are just from `go fmt`.

File: main.go [hunk 3]
```diff
@@ -59,26 +64,41 @@ func main() {
...
 	fmt.Println("DATABASE_URL", config.Database)
 
-	initAwsBucket()
-	initDB()
+	initRollbar()
 
-	fmt.Println("Running on port", config.Port)
-	http.HandleFunc("/", handler)
-	http.ListenAndServe(":"+strconv.Itoa(config.Port), nil)
+	var finishSetupAndListen = func() {
```
Comment (xxx) [inline main.go:88]: splitting this into a fully separate function seemed unnecessary since it's kind of a hodgepodge

File: main.go [hunk 4]
```diff
@@ -120,11 +155,16 @@ func initAwsBucket() {
...
 	var err error
-	db, err = sql.Open("postgres", config.Database)
 
+	db, err = sql.Open("postgres", config.Database)
 	if err != nil {
-		log.Fatalf("Error opening database: %q", err)
+		panic(err)
 	}
+
+    err = db.Ping()
```
Comment (xxx) [inline main.go]: this is how you actually verify the connection

**Review by mikecx — APPROVED** [review]
Looks pretty good to me, definitely harder for me to judge go commits than it should be.

**Pull request merged**